x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -1520,29 +1515,8 @@ describe('Task Runner', () => {
         true
       );
       actionsClient.ephemeralEnqueuedExecution.mockResolvedValue(mockRunNowResponse);
-      ruleType.executor.mockImplementation(
-        async ({
-          services: executorServices,
-        }: RuleExecutorOptions<
-          RuleTypeParams,
-          RuleTypeState,
-          AlertInstanceState,
-          AlertInstanceContext,
-          string,
-          RuleAlertData
-        >) => {
-          executorServices.alertFactory.create('1').scheduleActions('default');
-          return { state: {} };
-        }
-      );
-
-      getSummarizedAlertsMock.mockResolvedValue({
-        new: {
-          count: 1,
-          data: [mockAAD],
-        },
-        ongoing: { count: 1, data: [] },
-        recovered: { count: 0, data: [] },
+      ruleType.executor.mockImplementation(async () => {
```

Comment:
curious why we removed the `executorServices.alertFactory.create('1').scheduleActions('default');`. is it not important to the test?

Reply:
We don't need to create an alert in the rule executor as we test the summarized alerts and return a mock for them :)